hw/ip/lc_ctrl/doc/theory_of_operation.md

```diff
@@ -391,13 +392,16 @@ For better security, all the [life cycle control signals](#life-cycle-decoded-ou
 The active ON state for every signal is broadcast as `4'b1010`, while the inactive OFF state is encoded as `4'b0101`.
 For all life cycle signals except the escalation signal ESCALATE_EN, all values different from ON must be interpreted as OFF in RTL.
 In case of ESCALATE_EN, all values different from OFF must be interpreted as ON in RTL.
+To that end, it is strongly suggested to use the functions `lc_tx_test_true_strict()`, `lc_tx_test_true_loose()`, `lc_tx_test_false_strict()` and `lc_tx_test_false_loose()` in the `lc_ctrl_pkg`.
```

Comment:
Maybe strengthen this language? E.g. 'To that end the functions ... must be employed unless their is a strong reason not to. This reason must be documented and agreed at block sign-off'

Reply:
Yeah let me make that language stronger.

Reply:
It probably makes sense to go through the design and clean the endpoints up after M2.5.2.
I have noticed that not all endpoints use these functions yet.

Reply:
#19058